week2/new Program/index.js

```diff
@@ -0,0 +1,108 @@
+"use strict"
+/****  it's a program that do the following:
```

Comment:
Thanks for adding a spec, so a reviewer/reader can see what the code is supposed to do 😄